### PR TITLE
Add notes for Event Correction API

### PR DESCRIPTION
**Pull request by Ramdeprasad**

### CODE REVIEW

File: api-reference/event-management/undo-ingested-events.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+openapi: delete /events
+---
+
+## Query Parameters: Allowed filter combinations
+<Warning>Every single event matching the filters will be deleted</Warning>
```
Comment (karthick-vinod) [inline api-reference/event-management/undo-ingested-events.mdx:6]: All matching events will be considered for deletion

File: api-reference/event-management/undo-ingested-events.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<Info>Maximum events deleted per request is limited to 5. This behaviour will change in Event Correction API v2</Info>
```
Comment (karthick-vinod) [inline api-reference/event-management/undo-ingested-events.mdx:7]:
```suggestion
<Info>Maximum of 5 events will be deleted per request. This behavior might change in future</Info>
```

File: api-reference/event-management/undo-ingested-events.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+<Accordion title="Parameters">
+`account_id` : Unique identifier of the account corresponding to the event
+
+`id` : Unique identifier of an event generated by Togai for reference
+
+`event_id`: Identifier of an event provided by the user
+
+`event_source_time`: Timestamp of an event provided by the user
+
+`created_at` : Timestamp corresponding to the event ingestion time
```
Comment (karthick-vinod) [inline api-reference/event-management/undo-ingested-events.mdx:19]: Add these as <ResponseField>s

https://mintlify.com/docs/api-playground/components/api/response

File: api-reference/event-management/undo-ingested-events.mdx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+`account_id`, `id`
+
+`account_id`, `event_id`
+
+`account_id`, `event_source_time`
+
+`account_id`, `event_id`, `created_at`
+
+`account_id`, `event_id`, `event_source_time`
+</Accordion>
```
Comment (karthick-vinod) [inline api-reference/event-management/undo-ingested-events.mdx:36]: Fine for now. We need to come up with a better way to represent this info

File: openapi.yaml [hunk 5]
```diff
@@ -1914,108 +1914,24 @@ paths:
...
-        - $ref: '#/components/parameters/filter_event_id'
       responses:
         '200':
           description: Success response
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/EventsCorrectionResponse'
-              example:
+              examples:
```
Comment (karthick-vinod) [inline openapi.yaml:1932]: Check if examples is working fine in mintlify. If not, try changing it to `example`

File: api-reference/event-management/undo-ingested-events.mdx (hunk 4 above)
Comment (Ramdeprasad) [reply api-reference/event-management/undo-ingested-events.mdx:36]: Ok

File: openapi.yaml (hunk 5 above)
Comment (Ramdeprasad) [reply openapi.yaml:1932]: examples is working fine. example is not working at all

**Review by karthick-vinod — APPROVED** [review]

**Pull request merged**